### PR TITLE
1434 comms api outbound support bkp

### PR DESCRIPTION
**Pull request by gabriel0003**
## Changes

What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

### CODE REVIEW

File: packages/apps/spaces/.env.development [hunk 1]
```diff
@@ -7,8 +7,8 @@ SETTINGS_API_KEY=8b010f38-e5ca-4923-a62e-9f073c5c7dbf
 FILE_STORAGE_API_PATH=http://localhost:10001
 FILE_STORAGE_API_KEY=9eb87aa2-75e7-45b2-a1e6-53ed297d0ba8
 
-OASIS_API_PATH=http://localhost:8006
-OASIS_API_KEY=10a6747a-97cd-4a6c-bcf5-e4ee89a12567
+COMMS_API_PATH=http://localhost:8013
+COMMS_API_KEY=10a6747a-97cd-4a6c-bcf5-e4ee89a12567
```
Comment (tsearle) [inline packages/apps/spaces/.env.development:13]: could you add the following lines for webrtc?

NEXT_PUBLIC_TURN_SERVER=turn:127.0.0.1:3478?transport=tcp
NEXT_PUBLIC_TURN_USER=foo
NEXT_PUBLIC_TURN_CREDENTIAL=1234

I didn't add it to my branch as I wasn't touching spaces

File: packages/apps/spaces/components/ui-kit/molecules/email-timeline-item/EmailTimelineItem.tsx [hunk 2]
```diff
@@ -170,7 +112,7 @@ export const EmailTimelineItem: React.FC<Props> = ({
           <div>
             <EmailParticipants
               from={isToDeprecate ? sentBy : from}
```
Comment (tsearle) [inline packages/apps/spaces/components/ui-kit/molecules/email-timeline-item/EmailTimelineItem.tsx:114]: isToDeprecate flag still relevant given the migration to the new api is now complete?

File: packages/server/comms-api/routes/mail.go [hunk 3]
```diff
@@ -1,170 +1,111 @@
...
 
-type MailPostRequest struct {
-	Sender     string `json:"sender"`
-	RawMessage string `json:"rawMessage"`
-	Subject    string `json:"subject"`
-	ApiKey     string `json:"api-key"`
-	Tenant     string `json:"X-Openline-TENANT"`
-}
+func addMailRoutes(conf *c.Config, rg *gin.RouterGroup, mailService *s.MailService) {
+	rg.POST("/mail/send", func(c *gin.Context) {
```
Comment (tsearle) [inline packages/server/comms-api/routes/mail.go:17]: There should be an api key check for sending e-mail as well

File: packages/apps/spaces/.env.development (hunk 1 above)
Comment (gabriel0003) [reply packages/apps/spaces/.env.development:13]: done

File: packages/apps/spaces/components/ui-kit/molecules/email-timeline-item/EmailTimelineItem.tsx (hunk 2 above)
Comment (gabriel0003) [reply packages/apps/spaces/components/ui-kit/molecules/email-timeline-item/EmailTimelineItem.tsx:114]: Will clean after

File: packages/server/comms-api/routes/mail.go (hunk 3 above)
Comment (gabriel0003) [reply packages/server/comms-api/routes/mail.go:17]: nope. Identity is passed

**Review by tsearle — APPROVED** [review]

**Pull request merged**